test/integration/sanity/FileOptions.e2e.test.js

```diff
@@ -0,0 +1,69 @@
+// <reference types="Cypress" />
+describe('Preview File Options', () => {
+    const token = Cypress.env('ACCESS_TOKEN');
+
+    const fileIdDoc = Cypress.env('FILE_ID_DOC');
+    const fileIdVideo = Cypress.env('FILE_ID_VIDEO');
+    const fileIdMp3 = Cypress.env('FILE_ID_MP3');
+
+    const fileOptions = {
+        [fileIdDoc]: {
+            startAt: {
+                value: 2,
+                unit: 'pages'
+            }
+        },
+        [fileIdVideo]: {
+            startAt: {
+                value: 15,
+                unit: 'seconds'
+            }
+        },
+        [fileIdMp3]: {
+            startAt: {
+                value: 3,
+                unit: 'seconds'
+            }
+        }
+    };
+
+    /* eslint-disable */
+    const getPage = (pageNum) => cy.get(`.page[data-page-number=${pageNum}]`);
```

Comment:
If this is used in more than one file, we should probably share it between them somehow.